README.md

```diff
@@ -25,3 +25,16 @@
     1. Issues SHOULD be reviewed by the Operators to determine whether the rule should be changed or the player should be sanctioned.
 1. Only players may fill [roles](Roles/).
 1. If a proposal offers a choice between two or more options an Operator should call a [multi-vote](multi-vote.md).
+
+## [Projects](Projects/)
```

Comment:
This reverts the change I made in https://github.com/cryptotechguru/EDENomicon/commit/747b6f8863177a66339ce473f27b095b89e945e5